python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -621,8 +672,7 @@ def _safe_flush_xml(self, this_id):
         if not isType(obj, EmptyGangaObject):
             split_cache = None
 
-            has_children = (not self.sub_split is None) and\
-                    (self.sub_split in obj._data) and hasattr(obj, self.sub_split) and len(getattr(obj, self.sub_split)) > 0
+            has_children = hasattr(obj, self.sub_split) and getattr(obj, self.sub_split)
```

Comment:
Hmm, it probably should be otherwise non splittable objects should throw an exception here... I wonder why this doesn't crop up in testing?